src/index.ts

```diff
@@ -75,6 +76,7 @@ export {
   // Precomputed Client
   PrecomputedFlagsRequestParameters,
   EppoPrecomputedClient,
+  convertContextAttributesToSubjectAttributes,
```

Comment:
The configuration wire will only display attributes in the context attribute format, however users of the eppo client (sans bandits) will expect their subject attributes to be in the same format that they provided, so this helper method merges the `numericAttributes` and `categoricalAttributes` together